frontend/src/assetTypings.d.ts

```diff
@@ -2,22 +2,22 @@
 // without tsc type error
 
 declare module '*.svg?inline' {
```

Comment:
Can you explain briefly the idea of what assetTypings is?

Reply:
when you import svg files directly into tsx/ts file, tsc goes mad and start complaining about it doesn't have corresponding type declarations. So this is globally available file, where tsc will look into (as well in others in the project) to find type declarations